classes/ActionScheduler.php

```diff
@@ -97,6 +97,10 @@ public static function init( $plugin_file ) {
 		add_action( 'init', array( $admin_view, 'init' ), 0, 0 ); // run before $store::init()
 
 		require_once( self::plugin_path('functions.php') );
+
+		if ( defined( 'WP_CLI' ) && WP_CLI ) {
+			\WP_CLI::add_command( 'action-scheduler', 'ActionScheduler_WPCLI_Scheduler_command' );
```

Comment:
Would you mind removing `\`? `WP_CLI` is enough here.  Removing the slash has two cool things

1. Makes this 5.2 friendly
2. The slash is not needed, there is no namespace in this file.